### PR TITLE
PowerVS: Support newly added regions

### PR DESCRIPTION
**Pull request by hamzy**
There are a bunch of new regions added to PowerVS.

### CODE REVIEW

File: region.go [hunk 1]
```diff
@@ -30,6 +30,12 @@ func GetRegion(zone string) (region string, err error) {
 		region = "osa"
 	case strings.HasPrefix(zone, "mon"):
 		region = "mon"
+	case strings.HasPrefix(zone, "fra"):
+		region = "fra"
```
Comment (mkumatag) [inline region.go]: I don't see any entries for this one in this [table](https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-creating-power-virtual-server), is this still in staging or am I missing something here?

**Review by mkumatag — CHANGES_REQUESTED** [review]
As discussed lets add tests and also entries in the https://github.com/ppc64le-cloud/powervs-utils/blob/bc42a581099f1cdd286f7b5596a4d61b730ef761/region.go#L48

File: region.go (hunk 1 above)
Comment (hamzy) [reply region.go]: In another channel, I saw `fra04` and `fra05` mentioned, but then noticed they actually used `eu-de-`.  Therefore I removed it.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Pull request merged**